sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs

```diff
@@ -28,10 +28,19 @@ public partial class EventHubOptions : Microsoft.Azure.WebJobs.Hosting.IOptionsF
         public const string LeaseContainerName = "azure-webjobs-eventhub";
         public EventHubOptions() { }
         public int BatchCheckpointFrequency { get { throw null; } set { } }
-        public Azure.Messaging.EventHubs.Primitives.EventProcessorOptions EventProcessorOptions { get { throw null; } }
+        public Azure.Messaging.EventHubs.EventHubConnectionOptions ConnectionOptions { get { throw null; } set { } }
+        public string Identifier { get { throw null; } set { } }
```

Comment:
Not that this property is not on the processor options should we prefix it?

Reply:
Actually, would this seem out of place since we no longer reference "processor" generally?
Perhaps `InvokeProcessorAfterReceiveTimeout` should be changed to just `InvokeAfterReceiveTimeout`?
@jsquire thoughts on the naming here?

Reply:
hrmmm.   In this case, I'm wondering if `InvokeFunctionAfterReceiveTimeout`  would better communicate the semantics.   Essentially, this controls whether the binding pings the function when there are no events after the wait time, right?

Reply:
Yeah, I believe so. I was thinking that the Function part was somewhat implied as we don't reference it any of the other options, but I wouldn't be against including it for this option in particular. What do you think about the `Identifier` option? Should that be `ProcessorIdentifier` or maybe `LoggingIdentifier`?

Reply:
At some point in the near future, we'll be opening up identifiers for all client types.   I wonder if it makes more sense to add the distinction between `ProcessorIdentifier` and `PublishingIdentifier` or no.   If not, then I think `LoggingIdentifier` or `EventHubsClientIdentifier` would work.

Reply:
So are we thinking:
`ProcessorIdentifier` and `InvokeFunctionAfterReceiveTimeout`?

Reply:
LGTM